armi/utils/flags.py

```diff
@@ -233,7 +233,7 @@ def extend(cls, fields: Dict[str, Union[int, auto]]):
             :id: I_ARMI_FLAG_EXTEND0
             :implements: R_ARMI_FLAG_EXTEND
 
-            A class method to extend a ``Flag`` with a vector of provided additional ``fields``,
+            A method to extend a ``Flag`` with a vector of provided additional ``fields``,
```

Comment:
This is one of the instances where I think "class method" is the proper term. Just "method" isn't technically incorrect, it's just lest specific.

Reply:
Yup. Fixing.